share/lib/python/neuron/gui.py

```diff
@@ -79,7 +79,7 @@ def run(self) :
       self.fun()
       time.sleep(self.interval)
 
-if h.nrnversion(9) == '2' or h.nrnversion(8).find('mingw') > 0:
+if h.nrnversion(9) == '2' or h.nrnversion(8).find('mingw') > 0 or h.nrnversion(8).find('Windows'):
```

Comment:
Wow! That’s quite a hard to guess! I wonder if there would be any to test such thing under CI.